server/src/db/db.ts

```diff
@@ -1353,4 +1368,110 @@ export class Database {
 			return {type: 'failure', reason: `${e}`}
 		}
 	}
+
+	/**Get player achievements */
+	public async getAchievements(
+		playerId: string,
+	): Promise<DatabaseResult<AchievementData>> {
+		try {
+			const result = await this.pool.query(
+				`
+				SELECT achievement_id, goal_id, progress, completion_time
+				FROM user_goals
+				WHERE user_id = $1;
+				`,
+				[playerId],
+			)
+
```

Comment:
This needs a test, but we can wait on that until we have the branch set up so we can collaborate on this